yt_dlp/extractor/bilibili.py

```diff
@@ -188,22 +180,18 @@ def _real_extract(self, url):
                  r'<iframe[^>]+src="https://secure\.bilibili\.com/secure,([^"]+)"'],
                 webpage, 'player parameters'))['cid'][0]
         else:
-            if 'no_bangumi_tip' not in smuggled_data:
-                self.to_screen('Downloading episode %s. To download all videos in anime %s, re-run yt-dlp with %s' % (
-                    video_id, anime_id, compat_urlparse.urljoin(url, '//bangumi.bilibili.com/anime/%s' % anime_id)))
             headers = {
                 'Content-Type': 'application/x-www-form-urlencoded; charset=UTF-8',
                 'Referer': url
             }
             headers.update(self.geo_verification_headers())
 
-            js = self._download_json(
-                'http://bangumi.bilibili.com/web_api/get_source', video_id,
-                data=urlencode_postdata({'episode_id': video_id}),
-                headers=headers)
-            if 'result' not in js:
-                self._report_error(js)
-            cid = js['result']['cid']
+            json__INITIAL_STATE__ = self._parse_json(
```

Comment:
```py
inital_state = self._search_json(...)
```

Reply:
> 

ok, i will rename it.

Reply:
Not just rename. I was asking to use `_search_json` instead of `_parse_json(_search_regex(...))`

Reply:
> Not just rename. I was asking to use `_search_json` instead of `_parse_json(_search_regex(...))`

Oh, sorry, I will check it